client/src/app/pages/assessment-management/assessment-settings/assessment-settings-page.tsx

```diff
@@ -46,12 +48,16 @@ import { ConditionalTooltip } from "@app/components/ConditionalTooltip";
 import { useLocalTableControls } from "@app/hooks/table-controls";
 import { NotificationsContext } from "@app/components/NotificationsContext";
 import { formatPath, getAxiosErrorMessage } from "@app/utils/utils";
-import { Questionnaire } from "@app/api/models";
+import { Questionnaire, Thresholds } from "@app/api/models";
 import { useHistory } from "react-router-dom";
 import { Paths } from "@app/Paths";
 import { ImportQuestionnaireForm } from "@app/pages/assessment-management/import-questionnaire-form/import-questionnaire-form";
 import ConfirmDeleteDialog from "@app/components/ConfirmDeleteDialog/ConfirmDeleteDialog";
 import { ExportQuestionnaireDropdownItem } from "./ExportQuestionnaireDropdownItem";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+
+dayjs.extend(utc);
```

Comment:
That is a good candidate for putting in `client/src/index.tsx` to configure it once and the same way for the entire project.